### PR TITLE
add kernel columns

### PR DESCRIPTION
**Pull request by gaoteng-git**
Add the below 4 columns into kernel view's table.
image

### CODE REVIEW

File: tb_plugin/torch_tb_profiler/profiler/module_parser.py [hunk 1]
```diff
@@ -181,12 +181,18 @@ def parse_kernels(kernel_list):
             for kernel in kernel_list:
                 dur = kernel.end_time - kernel.start_time
                 op_name = "N/A" if kernel.op_node is None else kernel.op_node.name
-                key = kernel.name + "###" + op_name
+                key = kernel.name + "###" + op_name + "###" + \
```
Comment (guotuofeng) [inline tb_plugin/torch_tb_profiler/profiler/module_parser.py:184]: use string.join instead of add operation?

File: tb_plugin/torch_tb_profiler/profiler/node.py [hunk 2]
```diff
@@ -166,6 +171,10 @@ def get_node_argument(event):
         if event.type == EventTypes.KERNEL:
             kwargs["blocks_per_sm"] = event.args.get("blocks per SM", 0)
             kwargs["occupancy"] = event.args.get("est. achieved occupancy %", 0)
+            kwargs["grid"] = str(event.args.get("grid"))
```
Comment (guotuofeng) [inline tb_plugin/torch_tb_profiler/profiler/node.py:174]: In case of no these fields, the value will be "None" string. Should we not include these values?

File: tb_plugin/test/test_profiler.py [hunk 3]
```diff
@@ -1644,5 +1644,136 @@ def test_memory_view(self):
...
+            "name": "op2", "pid": 13721, "tid": "456",
+            "ts": 340, "dur": 70,
+            "args": {"Input Dims": [[], [32, 1000], [32], [], [], [], []], "External id": 4}
+          },          
+          {
+            "ph": "X", "cat": "Kernel", 
+            "name": "kernel1", "pid": 0, "tid": "stream 7",
+            "ts": 230, "dur": 15,
+            "args": {"correlation": 1000, "external id": 3, "device": 0,
+                    "grid": [16, 1, 1], "block": [16, 16, 16], "registers per thread": 18, "shared memory": 0}
```
Comment (guotuofeng) [inline tb_plugin/test/test_profiler.py:1668]: should we add test cases for missing these values?

**Review by guotuofeng — CHANGES_REQUESTED** [review]

File: tb_plugin/torch_tb_profiler/profiler/module_parser.py (hunk 1 above)
Comment (gaoteng-git) [reply tb_plugin/torch_tb_profiler/profiler/module_parser.py]: Thanks. Done.

File: tb_plugin/torch_tb_profiler/profiler/node.py (hunk 2 above)
Comment (gaoteng-git) [reply tb_plugin/torch_tb_profiler/profiler/node.py]: Got it. But I prefer still keep the kernel there, and with "None" as this field's value.
I've explicitly set the default value as "None" in latest commit.

File: tb_plugin/test/test_profiler.py (hunk 3 above)
Comment (gaoteng-git) [reply tb_plugin/test/test_profiler.py:1668]: Manually removed a kernel's "block" field, and test pass on UI side:
image

Comment (guotuofeng) [reply tb_plugin/test/test_profiler.py:1668]: It would be better to make the edge case automatically

File: tb_plugin/torch_tb_profiler/profiler/node.py [hunk 4]
```diff
@@ -166,6 +171,10 @@ def get_node_argument(event):
         if event.type == EventTypes.KERNEL:
             kwargs["blocks_per_sm"] = event.args.get("blocks per SM", 0)
             kwargs["occupancy"] = event.args.get("est. achieved occupancy %", 0)
+            kwargs["grid"] = str(event.args.get("grid", "None"))
```
Comment (guotuofeng) [inline tb_plugin/torch_tb_profiler/profiler/node.py:174]: keep the original value by removing the str

File: tb_plugin/test/test_profiler.py (hunk 3 above)
Comment (gaoteng-git) [reply tb_plugin/test/test_profiler.py:1668]: Done. Added a unit test handling kernel without some field.

File: tb_plugin/torch_tb_profiler/profiler/node.py (hunk 4 above)
Comment (gaoteng-git) [reply tb_plugin/torch_tb_profiler/profiler/node.py]: Thanks, done.

**Review by guotuofeng — APPROVED** [review]
